zoom_rooms/indico_zoom_rooms/tasks.py

```diff
@@ -0,0 +1,115 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2024 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from pprint import pformat
+
+import requests
+from celery.schedules import crontab
+
+from indico.core.celery import celery
+from indico.core.db import db
+from indico.core.plugins import current_plugin
+from indico.util.string import strip_control_chars
+
+from indico_zoom_rooms.models import ZoomRoomsAction, ZoomRoomsQueueEntry
+
+
+def _send_request(method: str, zr_id: str, entry_id: str, data: dict | None = None) -> bool:
+    """Send a request to the corresponding HTTP service."""
+    from indico_zoom_rooms.plugin import ZoomRoomsPlugin
+    logger = current_plugin.logger
+
+    path = f'/api/v1/users/{zr_id}/events/{entry_id}'
+    if not (url := ZoomRoomsPlugin.settings.get('service_url')):
+        raise KeyError('service_url is not set!')
+    url = url.rstrip('/') + path
```

Comment:
Not tested, but maybe it's cleaner to use urllib for this? Assuming you never expect another path in the service URL of course.

```python
url = urlunsplit(urlsplit(url)._replace(path=path))
```

Reply:
Actually I see we do the same in the outlook plugin, so let's keep it simple here as well